### ISSUE TITLE
Add the capability of reading a configuration file like "conf.lua"

### ISSUE THREAD

**Issue by cauehenrique** [morew4rd/lyte2d#6]
### TLDR
A great addition (and a very important one in my opinion) is to make **Lyte2D** able to read an external configuration file like `conf.lua` for pre-load settings such as `window_width`, `window_height`.

### Problem
I started playing with **Lyte2D** and while making my game I noticed that the window always _pops_ on the top-left side of the screen with it's default size even when I change it on the `app.lua` file. The same occurs on **Love2D** if you put stuff on `love.load` and their solution for that is by handling a configuration file before the game startup.

My `app.lua` file:
```lua
lyte.set_window_size(640, 360)
lyte.set_default_filtermode("nearest")

-- Rest of the code...
```

`config.lua` could be something like this:
```lua
return {
  window_width = 640,
  window_height = 360,
  loading_screen_enabled = false,
  default_filtermode = "nearest",

  -- Rest of the code...
}
```

Also I would love to disable the rotating "wood plank" loading screen (make it white or `loading_screen_color`).

**Comment by morew4rd** [comment]
Very good points! This is needed, but there are a few design considerations especially around packing up the game and single file distribution.

- `config.lua` file should be kept next to the exe/archive, not in the exe/archive. I.e. when packing up/fusing a distributable file, a copy of config should be left outside.
- If `config.lua` is not found what are the options? (1) Either open a window with some default size, or (2) wait for the whole archive to download and let `app.lua` tell Lyte2D what the window size should be. (2) can fail or take a long time. During which the gamer may think the app crashed or something.  So, I'm leaning towards (1)

Re loading screen: I'm thinking that config.lua can supply a "loading_tick" function in which the user can draw on their own, perhaps a logo or something. The problem with the previous paragraph applies here too, though. Since the full game/app is not loaded at this point, a logo for example will need to be left outside of the zip/exe archive. Perhaps we can make this a "hidden feature"

I'll think a bit more about this.

**Comment by morew4rd** [comment]
Forgot to add: I should also think about how errors show on the window as well. Love2D did a great job, perhaps I can copy some of that.

**Comment by morew4rd** [comment]
I made the first attempt and made a "pre-release" of v0.6: https://github.com/morew4rd/lyte2d/releases/tag/v0.6-pre.6. Feel free to give it a download and try!

You can create a `config.lua` file next to your app.lua/app.zip (in an unfused app) and next to your app.exe (in a fused app) to take effect. If none found, some defaults are applied.

Here's an example: https://github.com/morew4rd/lyte2d/blob/main/games/pongout/config.lua 

Small progress on the error messages front too, but really #3 is tracking that. Also, the default "loading" screen is changed and you can override it in config.lua like showed above. 

Any other thoughts on this one?

**Comment by morew4rd** [comment]
Fixed in 0.6 (https://github.com/morew4rd/lyte2d/releases/tag/v0.6)

Please file another one/reopen if you're hitting issues on this one.
